draft-ietf-quic-tls.md

```diff
@@ -1519,6 +1584,51 @@ SHOULD track redundant packets and treat excessive volumes of any non-productive
 packets as indicative of an attack.
 
 
+## Packet Number Protection Analysis {#pn-encrypt-analysis}
+
+Packet number protection relies on the randomness of the AEAD output, which is a
+property that AEAD algorithms do not guarantee.  Therefore, no strong assurances
+about the general security of this mechanism can be proven.
+
+Use of the same key and nonce for encryption for encryption can weaken
+encryption.  For the schemes described, protecting two packet numbers with the
+same key and nonce would reveal the packet number.  For packet number protection
```

Comment:
for encryption twice?

Reply:
Should I say two "different" packet numbers?  Basically, this is the standard admonition against reuse of key and nonce.